Uni_life_tracker/app/src/main/res/layout/activity_add_expenses.xml

```diff
@@ -0,0 +1,161 @@
+            android:layout_height="wrap_content"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintLeft_toLeftOf="parent"
+            app:layout_constraintRight_toRightOf="parent"
+            app:errorEnabled="true">
+    </com.google.android.material.textfield.TextInputLayout>
+
+    <com.google.android.material.textfield.TextInputEditText
+            android:layout_width="240dp"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintLeft_toLeftOf="parent"
+            app:layout_constraintRight_toRightOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            android:layout_height="wrap_content"
+            android:hint="@string/add_name"
+            android:inputType="text"
+            app:layout_constraintHorizontal_bias="0.187" app:layout_constraintVertical_bias="0.288"
+            android:id="@+id/add_e_name"/>
+    <com.google.android.material.textfield.TextInputLayout
+            android:id="@+id/e_add_amount"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintLeft_toLeftOf="parent"
+            app:layout_constraintRight_toRightOf="parent"
+            app:errorEnabled="true">
+    </com.google.android.material.textfield.TextInputLayout>
+
+
+    <com.google.android.material.textfield.TextInputEditText
+            android:layout_width="240dp"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintLeft_toLeftOf="parent"
+            app:layout_constraintRight_toRightOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            android:layout_height="wrap_content"
+            android:hint="@string/dollar_sign"
+            android:inputType="numberDecimal"
+            app:layout_constraintHorizontal_bias="0.187"
+            app:layout_constraintVertical_bias="0.583"
+            android:id="@+id/add_e_amount"/>
+    <com.google.android.material.textfield.TextInputLayout
+            android:id="@+id/e_add_date"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintLeft_toLeftOf="parent"
+            app:layout_constraintRight_toRightOf="parent"
+            app:errorEnabled="true">
+    </com.google.android.material.textfield.TextInputLayout>
+
+
+    <com.google.android.material.textfield.TextInputEditText
+            android:layout_width="240dp"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintLeft_toLeftOf="parent"
+            app:layout_constraintRight_toRightOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            android:layout_height="wrap_content"
+            android:hint="@string/add_date"
```

Comment:
I think we should name this something else because it doesn't make sense to have a date for expense

Reply:
I can take out date entirely